pkgs/development/compilers/llvm/git/clang/default.nix

```diff
@@ -71,7 +71,7 @@ let
       ln -sv $out/bin/clang $out/bin/cpp
 
       mkdir -p $lib/lib/clang
-      mv $lib/lib/${lib.versions.major version} $lib/lib/clang/${lib.versions.major version}
+      mv $lib/lib/19 $lib/lib/clang/19
```

Comment:
I'd actually prefer `lib.versions.major version`, which of course only work for versions like `19.0.0-unstable-2024-03-16`.

Reply:
Fixed